ignite/metrics/mean_average_precision.py

```diff
@@ -0,0 +1,422 @@
+from collections import defaultdict
+from typing import Any, Callable, Dict, List, Optional, Set, Tuple, Union
+
+import torch
+
+from ignite.metrics.metric import Metric, reinit__is_reduced, sync_all_reduce
+
+__all__ = ["MeanAveragePrecision"]
+
+
+def _iou(y: torch.Tensor, y_pred: torch.Tensor, crowd: List) -> torch.Tensor:
```

Comment:
Does this function tested ?

Reply:
Compare it against pycocotools iou function in test?